### DISCUSSION TITLE
Zoraxy update?

### DISCUSSION THREAD

**Discussion by sbminer123** [community-scripts/ProxmoxVE#1090] [Q&A] [answered]
I would like to ask about the Zoraxy server, how would I go about updating that?

**Comment by oOStroudyOo** [comment]
While the website says it's not updateable, I'm just reading the code and it looks like it may be.

Have you trying running `update` in your Zoraxy LXC?
If that doesn't work try `bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/zoraxy.sh)"`

Let me know if it works and we can update the website.

**Reply by sbminer123** [reply, marked as answer]
Hi yea sorry for the delay, Looks like zoraxy does support the "Update" command. Thank you.

**Comment by rabbit6984** [comment]
Hi i have still the Problem with the Update from 3.1.1 to 3.16
Zoraxy LXC with helper Script to 3.1.1 now i want to make the Update with the update Command or the zoraxy.sh where linked.
On the shell he said Updated Zoray the Zoraxy_version under /opt say 3.1.6 but on the WEbinterface still say 3.1.1 ?

Reboot / BrwoserCache i have testet without success.

Shell:
![image](https://github.com/user-attachments/assets/366c7b16-ee06-4e8a-a966-4a72818eddf3)

Webinterface
![image](https://github.com/user-attachments/assets/211bd371-2ad2-4299-8d34-d4a7ddcfe602)

Can someone help me?
Thanks

**Reply by vincenzobpt** [reply]
the same issue with my LXC installation, did you find any solution?
Thanks

**Reply by tremor021** [reply]
do:
```
systemctl stop zoraxy
rm -f /opt/zoraxy/zoraxy
update
```

**Reply by tremor021** [reply]
you can also try clearing your browser cache
